.github/sam.yaml

```diff
@@ -0,0 +1,18 @@
+AWSTemplateFormatVersion: '2010-09-09'
+Transform: AWS::Serverless-2016-10-31
+Description: >
+  deno-runtime
+
+  Example custom runtime for Deno
+  
+Resources:
+  DenoRuntime:
+    Type: AWS::Lambda::LayerVersion
+    Properties:
+      Content:
+        S3Bucket: begin-public-east-1
+        S3Key: deno-runtime.zip
```

Comment:
It may be the s3 thing is not required (and can use ContentUri): https://serverless.pub/sar-layers/

Reply:
> Is this deno-lambda-layer.zip ?

yes

> It may be the s3 thing is not required (and can use ContentUri): https://serverless.pub/sar-layers/

maybe! seems the docs are pretty thin and maybe in flux; typical aws! =P

Reply:
Didn't work without s3! AWS docs seem very thin :)